public/views/groupsView.js

```diff
@@ -122,8 +124,19 @@ export default class GroupsView extends BaseView {
 			Router.go('/groups', false);
 		})
 
+		this.newGroupWindow.addEventListener('click', () => {
+			let errField = document.getElementById(NewGroupConst.inputInfo.jsIdError);
+			errField.textContent = "";
+		}, true);
+
+
 		if (this._addGroupBtn !== null) {
 			this._addGroupBtn.addEventListener('click', () => {
+                if (this._titleField.value.trim() === "") {
+					let errField = document.getElementById(NewGroupConst.inputInfo.jsIdError);
+					errField.textContent = "Это поле не может быть пустым";
```

Comment:
```suggestion
					errField.textContent = "Заполните это поле";
```
фокус размывается иначе, у нас тут призыв к конкретному действию